### PR TITLE
mk-feature-1 

### PR DESCRIPTION
**Pull request by kamir**
- added changed files for new feature - read from textfile via URL

### CODE REVIEW

**Comment by 2pk03** [comment]
@kamir - Build throws an error: https://github.com/apache/incubator-wayang/actions/runs/10364302090/job/28689337514?pr=459#step:7:32998

Can you please check?

File: wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala [hunk 1]
```diff
@@ -112,6 +123,7 @@ class JavaPlanBuilder(wayangCtx: WayangContext, jobName: String) {
     */
   def withUdfJar(path: String) = {
     this.planBuilder withUdfJars path
+    System.out.println("**MK** ---MARKER--- in JavaPlanBuilder" )
```
Comment (novatechflow) [inline wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala]: Please change this into a more understandable error message.

File: wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala [hunk 2]
```diff
@@ -135,6 +147,7 @@ class JavaPlanBuilder(wayangCtx: WayangContext, jobName: String) {
     */
   def withUdfJarOf(cls: Class[_]) = {
     this.planBuilder withUdfJarsOf cls
+    System.out.println("**MK** ---MARKER--- in JavaPlanBuilder" )
```
Comment (novatechflow) [inline wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala]: same ;)

**Comment by kamir** [comment]
Cleaning the code .... removed personal comments.

File: wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala (hunk 2 above)
Comment (kamir) [reply wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala]: Done.

File: wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala (hunk 1 above)
Comment (kamir) [reply wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala]: Done.

**Comment by kamir** [comment]
> @kamir - Build throws an error: https://github.com/apache/incubator-wayang/actions/runs/10364302090/job/28689337514?pr=459#step:7:32998
> 
> Can you please check?

Check Done.

- FAILS in cloud: 

[INFO] 
Error:  Errors: 
Error:    TensorflowIrisIT.test:153 » Wayang Executing T[JavaCollect[convert out@JavaMap...

+ at home in my branch  "kamir:mk-feature-1" the tests pass.

**Review by 2pk03 — APPROVED** [review]
Great contribution, thank you @kamir!

**Pull request merged**